setup.py

```diff
@@ -37,6 +37,7 @@
     entry_points={
         'console_scripts': [
             'run-ci=ocs_ci.run_ocsci:main',
+            'report-version=ocs_ci.ocs.version:main',
```

Comment:
I believe we want this by default for all runs?

Reply:
Yes, and the code as it is is integrated in this way. But by having this script, it's also possible to run this anytime later.

Reply:
I see the link now, It make sense, resolving both comments.